### PR TITLE
Task usage metrics documentation

### PR DESCRIPTION
**Pull request by jwitz**
Already reviewed in https://github.com/astronomer/docs/pull/1462, just adding here to make for a cleaner PR

### CODE REVIEW

File: software/task-usage-metrics.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    ```text
+    https://github.com/astronomer/airflow-analytics-plugin/releases/download/1.0.3/astronomer_analytics_plugin-1.0.3-py3-none-any.whl
+    ```
+
+2. Push your changes to Astronomer Software. See [Deploy code](deploy-cli.md).
+
+## View task usage metrics
+
+Depending on your permissions, you can view task usage metrics for a given installation, Workspace, or Deployment in the Astronomer UI.
```
Comment (kwcanuck) [inline software/task-usage-metrics.md:37]:
```suggestion
Depending on your permissions, you can view task usage metrics for a given installation, Workspace, or Deployment in the Astronomer Software UI.
```

Comment (kwcanuck) [reply software/task-usage-metrics.md:37]: Should the other instances of _Astronomer UI_ be updated as well?

**Review by kwcanuck — APPROVED** [review]

**Review by kwcanuck — APPROVED** [review]

File: software/release-notes.md [hunk 2]
```diff
@@ -76,6 +76,14 @@ Release date: December 23, 2022
 
 Release date: December 7, 2022
 
+### View and export task usage metrics
+
+You can now view and export task usage metrics from the Software UI.
```
Comment (rishkarajgi) [inline software/release-notes.md:81]: Export feature doesn't exist, so it's only 'view'

File: software/task-usage-metrics.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+```yaml
+global:
+  taskUsageMetricsEnabled: true
+```
+
+After you set this configuration, Astronomer Software begins to run a cron job for collecting task usage metrics at the 57th minute of every hour, sending them to Astronomer support, and displaying them in the Software UI every. By default, your Astronomer installation retains this data for 90 days. 
+
+## Set up task usage metrics on a Deployment (_Required only for Astronomer Certified and Astro Runtime < 7.2_)
+
+Task usage metrics are generated by the Astronomer-maintained `airflow-analytics-plugin`. This plug-in queries the Airflow metadata database and returns usage analytics at an HTTP endpoint. Deployments using Runtime 6.1 or later have this plug-in installed by default.
```
Comment (rishkarajgi) [inline software/task-usage-metrics.md:23]: 6.1 -> 7.2

File: software/release-notes.md (hunk 2 above)
Comment (jwitz) [inline software/release-notes.md:81]:
```suggestion
You can now view task usage metrics from the Software UI.
```

File: software/task-usage-metrics.md (hunk 3 above)
Comment (jwitz) [inline software/task-usage-metrics.md:23]:
```suggestion
Task usage metrics are generated by the Astronomer-maintained `airflow-analytics-plugin`. This plug-in queries the Airflow metadata database and returns usage analytics at an HTTP endpoint. Deployments using Runtime 7.2 or later have this plug-in installed by default.
```

**Pull request merged**